torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -216,6 +218,42 @@ def get_default_x86_inductor_quantization_config(is_qat: bool = False):
     return quantization_config
 
 
+@functools.lru_cache
+def get_default_x86_inductor_dynamic_quantization_config():
+    act_quantization_spec = QuantizationSpec(
+        dtype=torch.uint8,
+        quant_min=0,
+        quant_max=255,
+        is_dynamic=True,
+        observer_or_fake_quant_ctr=PlaceholderObserver.with_args(is_dynamic=True),
```

Comment:
this is PTQ only? maybe you can add this to the name of this util

Reply:
Thanks. I have added QAT support.